cmake/FindSIP.cmake

```diff
@@ -0,0 +1,91 @@
+# Find SIP
+# ~~~~~~~~
+#
+# SIP website: http://www.riverbankcomputing.co.uk/sip/index.php
+#
+# Find the installed version of SIP. FindSIP should be called after Python
+# has been found.
+#
+# This file defines the following variables:
+#
+# SIP_VERSION - SIP version.
+#
+# SIP_EXECUTABLE - Path to the SIP executable.
+#
+# SIP_INCLUDE_DIRS - The SIP include directories.
+#
+
+# Copyright (c) 2007, Simon Edwards <simon@simonzone.com>
+# Redistribution and use is allowed according to the terms of the BSD license.
+# For details see the accompanying COPYING-CMAKE-SCRIPTS file.
+
+if(APPLE)
+    # Workaround for broken FindPythonLibs. It will always find Python 2.7 libs on OSX
+    set(CMAKE_FIND_FRAMEWORK LAST)
+endif()
+
+# FIXME: Use the new FindPython3 module rather than these. New in CMake 3.12.
+# However currently that breaks on our CI server, since the CI server finds the built-in Python3.6 and then doesn't find the headers.
+find_package(PythonInterp 3.4 REQUIRED)
```

Comment:
shouldn't this be version 3.5 ?

Reply:
Yes, we should increment this. I copied this from libSavitar so we should also fix it there and possibly on libArcus too. Cura 15.6 started off with Python 3.4, which is probably where this comes from.